localization/util/autoware_localization_srvs/package.xml

```diff
@@ -17,6 +17,9 @@
 
   <member_of_group>rosidl_interface_packages</member_of_group>
 
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_cmake_cppcheck</test_depend>
```

Comment:
Since this package doesn't have any C++, maybe this should be changed to `ament_lint_common`.

Reply:
@jilaada fixed it.